test-cases/test-cases.go

```diff
@@ -672,6 +695,33 @@ func runTests(tests []*PackageTests) (int, *bytes.Buffer, error) {
 	return exitCode, &errorOutput, nil
 }
 
+func generateTAPOutput(tests []*PackageTests) {
+	i := 0
+	prefix := findCommonPrefix(tests)
+	for _, p := range tests {
+		pkgName := p.Name[len(prefix):]
+		fmt.Printf("# Tests for %s\n", pkgName)
+		for _, t := range p.Tests {
+			if t.Result == "PASS" {
+				fmt.Printf("ok ")
+			} else {
+				fmt.Printf("not ok ")
+			}
+			testName := strings.TrimSpace(t.Summary)
+			if testName == "" {
+				testName = t.Name
+			} else {
+				testName = newLineRegexp.ReplaceAllString(testName, " ")
+			}
+			fmt.Printf("%d - %s\n", i+1, testName)
+			i++
+		}
+	}
+	if i > 0 {
+		fmt.Printf("1..%d\n", i)
+	}
```

Comment:
After is okay as well.  Outputting at the bottom here as this is what the release_test.py script does.  The goal is to replace this script without breaking the testing infrastructure built around it.